tests/bud/target/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM fedora:latest
+RUN touch /foo
+
+FROM alpine:latest AS mytarget 
```

Comment:
Maybe add one more stage after this to ensure that it is skipped when --target it set to mytarget. To ensure we don't accidentally regress in future code changes.